en/mkdocs.yml

```diff
@@ -28,6 +28,15 @@ dev_addr: localhost:8000
 # Copyright
 copyright: Copyright © WSO2 LLC (2022-2023)
 
+#Plugins
+plugins:
+  - glightbox
```

Comment:
Why do we need to add this plugin?

Reply:
This is required to make the top-level items of the left nav clickable.